engines/freescape/games/dark.cpp

```diff
@@ -0,0 +1,152 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "common/config-manager.h"
+#include "common/file.h"
+
+#include "freescape/freescape.h"
+#include "freescape/language/8bitDetokeniser.h"
+
+namespace Freescape {
+
+DarkEngine::DarkEngine(OSystem *syst, const ADGameDescription *gd) : FreescapeEngine(syst, gd) {
+	_viewArea = Common::Rect(40, 24, 279, 124);
+	_playerHeightNumber = 1;
+	_playerHeights.push_back(16);
+	_playerHeights.push_back(48);
+
+	_playerHeight = _playerHeights[_playerHeightNumber];
+	_playerHeight = 64;
+	_playerWidth = 12;
+	_playerDepth = 32;
+}
+
+void DarkEngine::loadAssets() {
+	Common::SeekableReadStream *file = nullptr;
+	Common::String path = ConfMan.get("path");
```

Comment:
Why are you doing all of this?

drop `path`, drop `gameDir`. write `file.open("DSIDEE.EXE")`. That's it. 

The check will be `if (file.isOpen())`

Reply:
Fixed.